test/scale-infra/delete-ec2.yaml

```diff
@@ -0,0 +1,44 @@
+---
+- hosts: localhost
+  gather_facts: no
+  vars:
+    region: "{{ region }}"
+    group: "{{ sig_group | default('ks-nodes') }}"
+    cluster_name: "{{ k8s_cluster_name | default('ks-core') }}"
+  tasks:
+  - fail: msg="cluster_name is required"
+    when: cluster_name is undefined
```

Comment:
Given that it has a default above, how could it ever be undefined?

Reply:
That was a typo! I removed that statement.